admin_app/src/app/urgency-rules/UDSidebar.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from "react";
```

Comment:
We have been putting components under `/component`. So suggest moving this to `urgency-rules/components`